src/schema/inventory/supplierItem/resolvers.js

```diff
@@ -1,10 +1,29 @@
 module.exports = {
-   Query: {},
+   Query: {
+      supplierItems: async (_, __, { models }) => {
+         try {
+            const { SupplierItem } = models
+            const supplierItems = await SupplierItem.find()
+            return supplierItems
+         } catch (error) {
+            return error.message
+         }
+      },
+      supplierItem: async (_, { id }, { models }) => {
+         try {
+            const { SupplierItem } = models
+            const supplierItem = await SupplierItem.find({ _id: id })
```

Comment:
```suggestion
            const supplierItem = await SupplierItem.findOne({ _id: id })
```
`find` returns an array, use `findOne` instead